apps/dav/lib/CalDAV/Schedule/IMipPlugin.php

```diff
@@ -178,12 +178,7 @@ public function schedule(Message $iTipMessage) {
 		$recipientName = $iTipMessage->recipientName ?: null;
 
 		if ($senderName === null || empty(trim($senderName))) {
-			$user = $this->userManager->get($this->userId);
-			if ($user) {
-				// getDisplayName automatically uses the uid
-				// if no display-name is set
-				$senderName = $user->getDisplayName();
-			}
+			$sender = $this->userManager->getDisplayName($this->userId) ?? $this->userId;
```

Comment:
```suggestion
			$sender = $this->userManager->getDisplayName($this->userId);
			if ($sender !== null) {
				// getDisplayName automatically uses the uid
				// if no display-name is set
				$senderName = $sender;
			}
```

It's a weird edge case, in case the sender would not exist.
If you don't care and want to keep it simple at least the variable name needs fixing:
```suggestion
			$senderName = $this->userManager->getDisplayName($this->userId) ?? $this->userId;
```

Reply:
no it's fine, I will remove the `??` for the userId.

It will check both variables further down anyway.